repose-aggregator/src/docs/asciidoc/recipes/docker.adoc

```diff
@@ -1,130 +1,147 @@
 
-    * *docker build* - This is the part of the command that tells Docker to build the image.
-    * *--build-args REPOSE_VERSION=9.9.9.9* - This option defines the version of Repose that will be installed in the image.
-    The REPOSE_VERSION key will be given a value of 9.9.9.9 which can be used inside the Dockerfile.
-    It exists so that we can build Docker images for any version of Repose using a single Dockerfile.
-    * *-t my_repose_image* - This option defines the tag (in repository:tag format) to apply to the image.
-    * */path/to/* - This is the path to the Repose Dockerfile.
-    The Dockerfile itself must exist as a child of the provided directory.
-    Note that this directory will also be treated as the Docker build context.
-. If the Dockerfile was successfully built, a Docker image should be available within Docker.
-Run the following command to show all available images:
-+
-----
-docker images
-----
+If you don't already have a configuration, then there is a couple of paths you might want to take.
+The first is if you don't know how *Repose* works and therefore need to know what you can do, then visit the <<getting-started.adoc#,Getting Started>>.
+We all need to get started somewhere and this is a great place if it is your first time using *Repose*.
+If you are already familiar with using *Repose* and are just looking for a quick reference of the different files, the visit the <<../architecture/configuration.adoc#,Configuration>> page for more specific details.
+We also have a series of <<index.adoc#,Recipes>> that will walk you through some of our common use cases and their configurations.
 
-== Docker Container
-
-=== Running
-After you have acquired a Repose Docker image, you can run Repose by creating and starting a Docker container from that image.
-For example, the following command could be use:
+The following command will mount a volume containing a configuration and perform a couple other new options to the `run` command which are explained below.
 
 ----
-docker run                             \ <1>
-   -d                                  \ <2>
-   -v /my/config/directory:/etc/repose \ <3>
-   -p 8080:8080                        \ <4>
-   -e JAVA_OPTS=-Xmx1024m              \ <5>
-   --name my_repose                    \ <6>
-   rackerlabs/repose:latest              <7>
+docker run                                   \ <1>
+   --detach                                  \ <2>
+   --volume /my/config/directory:/etc/repose \ <3>
+   --publish 8080:8080                       \ <4>
+   --env JAVA_OPTS=-Xmx1024m                 \ <5>
+   --name my_repose                          \ <6>
+   rackerlabs/repose:latest                    <7>
 ----
 Let's break that command down and take a closer look at what it is doing:
 
 <1> This is the part of the command that tells Docker to create and start a container.
-<2> This flag will run the container in detached mode.
+<2> This option will run the container in detached mode.
```

Comment:
Not that it makes much difference, but why the change here?

Reply:
The `man` pages call them `option`'s.

Reply:
Fair enough. `option`s are the general term, right? And `flag`s are generally accepted to be boolean `option`s? That's how I've always used them, but being consistent with the `man` pages is totally legit.